README.md

```diff
@@ -36,3 +36,157 @@ This is the shared library defining public classes that constitute the worker in
 ### worker-document-testing
 
 This contains implementations of the testing framework to allow for integration testing of the Document Worker. The project can be found in [worker-document-testing](worker-document-testing).
+
+## Workflow event handlers
+
+The below are the various event handlers from the workflow scripts.
+
+It is not mandatory that the workflow script file should contain all of the below event handlers.
+Sometimes the workflow script may contain only few handlers from the below list.
+The event handler will only be triggered, if there is corresponding function from the JavaScript file being executed.
+
+####  onProcessTask
+
+This is the first function called by worker on the task message.
+This function is passed `TaskEventObject` with `Task` as an argument.
+
+An example of the `TaskEventObject` is below. The variables will be initialized with values from task message and it will be sent to the series of other functions in the workflow for further processing.
```

Comment:
Remove The variables will be initialized with values from task message and it will be sent to the series of other functions in the workflow for further processing.

Reply:
Removed